core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -178,7 +185,11 @@ class InvokerPool(childFactory: (ActorRefFactory, InvokerInstanceId) => ActorRef
     }
   }
 
-  /** Pads a list to a given length using the given function to compute entries */
+  /**
+   * Pads a list to a given length using the given function to compute entries
+   * Essentially this functions adds dummy values for the Invokers whose
+   * Ping messages haven't arrived yet
```

Comment:
Please pull the second part of the comment to the line where `padToIndexed` is used, since that's where the "essentially this function adds dummy values" becomes significant.

`padToIndexed` itself is a pretty general method.